daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/LF.daml

```diff
@@ -128,16 +128,17 @@ instance Action Update where
 instance ActionFail Update where
     fail = primitive @"UAbort"
 
--- | Places where the time is available - basically `Scenario` and `Update`.
+-- | The `HasTime` class is for where the time is available: `Scenario` and `Update`.
 class HasTime m where
```

Comment:
@neil-da Should we add a `Action m` super class constraint? Then the documentation could be:
The `HasTime` class is for `Action`s that support a notion of current time, like `Update` and `Scenario`.